lib/card_generator.rb

```diff
@@ -0,0 +1,19 @@
+require './lib/card'
+class Card_Generator
+  attr_reader :cards
+  def initialize(filename)
+    @filename = filename
+    file = File.open(filename)
+    file_data = file.readlines.map.each {|x| x.split(",")}
```

Comment:
Make sure to use semantic naming for your variables. Avoid using variables like `x`  or `y`